lib/svg-sprite.js

```diff
@@ -383,20 +384,20 @@ SVGSpriter.prototype._logStats = function(files) {
     const sizes = {};
     const exts = {};
 
-    for (const mode in files) {
-        for (const resource in files[mode]) {
-            const file = files[mode][resource].relative;
-            const ext = path.extname(files[mode][resource].path).toUpperCase();
+    for (const mode of Object.values(files)) {
+        for (const resource of Object.values(mode)) {
+            const file = resource.relative;
+            const ext = path.extname(resource.path).toUpperCase();
             exts[ext] = (exts[ext] || 0) + 1;
-            sizes[file] = pretty(files[mode][resource].contents.length);
+            sizes[file] = pretty(resource.contents.length);
         }
     }
 
     this.info('Created %s', Object.entries(exts).map(ext => `${ext[1]} x ${ext[0].substr(1)}`).join(', '));
 
-    Object.keys(sizes).sort().forEach(file => {
-        this.verbose('Created %s: %s', file, sizes[file]);
-    });
+    for (const [file, size] of Object.entries(sizes).sort()) {
```

Comment:
The sorting here is changed. But also I'm not sure of sorting here is needed

Reply:
Not sure either now that you mention. We could remove sorting. Feel free to push to this branch.

Reply:
Alternatively, we could try using something like:

```js
for (const [file, size] of Object.fromEntries(Object.entries(sizes).sort())) {
    this.verbose('Created %s: %s', file, size);
}
```